lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -576,8 +572,10 @@ export class PublicClientApplication {
      * @param authorityUri 
      */
     private async getClientConfiguration(authorityUri?: string): Promise<ClientConfiguration> {
-        const discoveredAuthority = authorityUri ? await AuthorityFactory.createDiscoveredInstance(authorityUri, this.config.system.networkClient) 
-            : await this.defaultAuthorityPromise;
+        const defaultAuthority = await this.defaultAuthorityPromise;
+        const shouldCreateAuthority = !!authorityUri && authorityUri !== defaultAuthority.canonicalAuthority;
```

Comment:
Are `authorityUri` and `defaultAuthority.canonicalAuthority` normalized somewhere? For example: Ensuring trailing slashes are consistent? Both are lower cased? etc.

Reply:
`defaultAuthority.canonicalAuthority` will always have a trailing slash, we should validate/normalize the given authorityUri however.

Reply:
verified that the authorities will always be consistent format and have trailing slashes.